monitoring/config.go

```diff
@@ -17,6 +20,9 @@ type PrometheusConfig struct {
 	// generic RPC metrics to monitor the health of the service.
 	RPCServer *grpc.Server
 
+	// UniverseStats ...
```

Comment:
nit: missing godoc

Reply:
It's added later on 6de7742db05861980b46079ccdc93e620a58b2a5 , could fixup?